server/services/caseloadService.ts

```diff
@@ -219,6 +219,8 @@ export default class CaseloadService {
       } else if (this.isRecall(offender)) {
         // Offender is subject to an active recall - not clickable
         licenceStatus = LicenceStatus.OOS_RECALL
+      } else if (offender.cvlFields.isInHardStopPeriod) {
```

Comment:
This is right assumption but think we need to make sure hardstop flag not enabled?